docs/usage/exporting_models.md

```diff
@@ -10,7 +10,10 @@ Arguments:
 * `include`: fields to include in the returned dictionary; see [below](#advanced-include-exclude)
 * `exclude`: fields to exclude from the returned dictionary; see [below](#advanced-include-exclude)
 * `by_alias`: whether field aliases should be used as keys in the returned dictionary; default `False`
-* `skip_defaults`: whether fields which were not explicitly set when creating the model should
+* `exclude_unset`: whether fields which were not explicitly set when creating the model should
+  be excluded from the returned dictionary; default `False`
+  * Prior to **v1.0**, `exclude_unset` was known as `skip_defaults`; use of `skip_defaults` is now deprecated
```

Comment:
This isn't working because you need 4 spaces for mkdocs (:gun:), but also I think better to just have one long sentence than a sub-point.